engine/src/main/java/org/destinationsol/game/sound/DebugHint.java

```diff
@@ -35,8 +37,11 @@
     private SolObject myOwner;
     private String myMsg;
 
-    public DebugHint(SolObject owner, Vector2 position) {
+    private EntityRef entity;
+
+    public DebugHint(SolObject owner, EntityRef entity, Vector2 position) {
```

Comment:
I feel like this would be appropriate as an overload so you don't have to send through arbitrary null values